restaurant/migrations/0001_initial.py

```diff
@@ -1,5 +1,4 @@
-# Generated by Django 3.1.2 on 2020-11-04 01:01
-
+# Generated by Django 3.1.2 on 2020-11-09 03:34
```

Comment:
This migration file is not consistent with what we have now. Could you try to pull the code from the develop branch first and then make migrations?

Reply:
Sure